content/contact/home.pml

```diff
@@ -46,27 +46,19 @@ The next sessions are:
 </p>
 <ul>
   <li>
-    Topic: seL4 boot code
+    Topics: feedback seL4 Core Platform
     <ul>
-      <li>Sydney: Wed, Nov 3, 8am</li>
-      <li>Central Europe: Tue, Nov 2, 10pm</li>
-      <li>US Pacific Time: Tue, Nov 2, 2pm</li>
-    </ul>
-  </li>
-  <li>
-    Topic: <i>open</i>
-    <ul>
-      <li>Sydney: Wed, Nov 17, 8am</li>
-      <li>Central Europe: Tue, Nov 16, 10pm</li>
-      <li>US Pacific Time: Tue, Nov 16, 2pm</li>
+      <li>Sydney: Wed, Dec 1, 8am</li>
+      <li>Central Europe: Tue, Nov 30, 10pm</li>
+      <li>US Pacific Time: Tue, Nov 30, 2pm</li>
```

Comment:
I knew I forgot to change the time somewhere. With daylight savings, the US has moved by 1h.

```suggestion
      <li>US Pacific Time: Tue, Nov 30, 1pm</li>
```